fedimint-api-client/Cargo.toml

```diff
@@ -34,6 +34,7 @@ fedimint-logging = { workspace = true }
 futures = { workspace = true }
 itertools = { workspace = true }
 jsonrpsee-core = { version = "0.24.7" }
+jsonrpsee-types = { version = "0.24.7" }
```

Comment:
Needed to introduce so we can use `METHOD_NOT_FOUND_CODE` to check for the error type instead of using string matching.